package.json

```diff
@@ -45,6 +45,7 @@
     "react": ">= 16.8.0"
   },
   "dependencies": {
+    "@chanzuckerberg/story-utils": "^3.0.4",
```

Comment:
Moved this because of a lint error that insists this is a main dependency, which doesn't seem right to me since the file it's pointing to is a test file 🤔 https://github.com/chanzuckerberg/edu-design-system/runs/5572780062

> /home/runner/work/edu-design-system/edu-design-system/src/components/AvatarImage/AvatarImage.spec.tsx
> Error:   1:1  error  '@chanzuckerberg/story-utils' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
> 
> ✖ 1 problem (1 error, 0 warnings)

Reply:
I just started running into this error too (not sure why it wasn't flagging before) but it's because it's a `spec` file not a `test` file 
![image](https://user-images.githubusercontent.com/86632227/158638606-d544017e-22a9-415a-a2e8-c98f0dc60dcc.png)